packages/kbn-storybook/templates/index.ejs

```diff
@@ -63,6 +63,7 @@
         <% for (var varName in globals) { %>
             <% if (globals[varName] != undefined) { %>
                 window['<%=varName%>'] = <%= JSON.stringify(globals[varName]) %>;
+                window.__kbnHardenPrototypes__ = false;
```

Comment:
@jbudz I think this has to go in the script block that starts on line 24 above. This variable has to be defined before the browser executes `kbn-ui-shared-deps-*`

Reply:
Whoops, yes thank you!  I pushed https://github.com/elastic/kibana/pull/209887/commits/d4ec6282580ae7de5f25649f993bbb6e4f1d54a8.